src/elements/content-sidebar/activity-feed/comment/components/BaseCommentMenuWrapper.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+import React from 'react';
+import type { BoxCommentPermission, FeedItemStatus } from '../../../../../common/types/feed';
+import type { OnAnnotationEdit, OnCommentEdit } from '../types';
+
+import { BaseCommentMenu } from './BaseCommentMenu';
+
+export interface BaseCommentMenuWrapperProps {
+    canDelete: boolean;
+    canEdit: boolean;
+    canResolve: boolean;
+    id: string;
+    isEditing: boolean;
+    isInputOpen: boolean;
+    isResolved: boolean;
+    onAnnotationEdit?: OnAnnotationEdit | typeof undefined;
+    onCommentEdit: OnCommentEdit;
+    onDelete: ({ id: string, permissions?: BoxCommentPermission }) => any;
+    onSelect: (isSelected: boolean) => void;
+    permissions: BoxCommentPermission;
+    setIsEditing: ((boolean => boolean) | boolean) => void;
+    setIsInputOpen: ((boolean => boolean) | boolean) => void;
+}
+
+export const BaseCommentMenuWrapper = ({
```

Comment:
In this case, this component is acting as a state container, so "Container" seems preferable.

Reply:
For definitions of wrappers versus containers, I've been going by the highest voted answer [here](https://stackoverflow.com/questions/4059163/css-language-speak-container-vs-wrapper) for a while. I've been thinking of Redux containers as a special convention outside of these definitions. What do you think about these definitions?

Reply:
Yeah, naming is always squishy. SUIT throws a wrench in here, as well, since `-` is use to indicate that something is a descendant. Ideally, components would start with a base name, like `Comment`, which would be the "wrapper" and everything inside it (e.g. `Comment-text`) would be descendants.

It's not a major issue either way, so long as the conventions are followed otherwise.